Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -13,19 +13,8 @@ public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceH
 		IDispatcher _dispatcher;
 		public virtual IDispatcher Dispatcher
 		{
-			get
-			{
-				if (_dispatcher == null)
-				{
-					_dispatcher = this.GetDispatcher();
-				}
-
-				return _dispatcher;
-			}
-			internal set
-			{
-				_dispatcher = value;
-			}
+			get => _dispatcher ??= this.GetDispatcher();
+			internal set => _dispatcher = value;
```

Comment:
[this comment applies to CanBeSetFromStyle; GitHub does not allow comments on "unchanged" code]
nit: give that you made some unrelated improvements in this file you can do one more. Instead of 3 IF checks you can use just one:

`
const BindableContextAttributes trueAttr =
		BindableContextAttributes.IsSetFromStyle |
		BindableContextAttributes.IsDefaultValue |
		BindableContextAttributes.IsDefaultValueCreated;
return ((context.Attributes & trueAttr) != 0);
`